hydromt/data_catalog.py

```diff
@@ -514,6 +514,7 @@ def get_rasterdataset(
         path_or_key: str,
         bbox: List = None,
         geom: gpd.GeoDataFrame = None,
+        zoom_level: Union[List, int] = 0,
```

Comment:
could you add zoom_level to the docstring here as well.

Reply:
Done